### PR TITLE
Civ 1107 under 18

### PR DESCRIPTION
**Pull request by TatsyMaslova**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-1107

### Change description ###

Added a new view for under 18's and logic to redirect to that page when the user is under 18.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/routes/urls.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ export const CASES_URL: Urls = '/cases';
 export const CITIZEN_PHONE_NUMBER_URL: Urls = '/citizen-phone';
 export const ROOT_URL: Urls = '/home';
 export const DOB_URL: Urls = '/your-dob';
+export const AGE_ELIGIBILITY: Urls = '/eligibility/under-18';
```
Comment (taomoh) [inline src/main/routes/urls.ts:11]: Add _URL suffix for consistency

Comment (TatsyMaslova) [reply src/main/routes/urls.ts:11]: good point! thank you Taoufik

File: src/main/common/utils/ageEligibilityVerification.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export const AGE_THRESHOLD = 18;
+export class AgeEligibilityVerification {
+  public static isOverEighteen(dob: Date) {
+    return AgeEligibilityVerification.getAge(dob) > AGE_THRESHOLD;
```
Comment (sampankumar) [inline src/main/common/utils/ageEligibilityVerification.ts:4]: Are we only checking for over Eighteen? If the claimant/defendant is 18 then what should happen?

**Review by taomoh — COMMENTED** [review]
LGTM. Just one minor comment

File: src/main/common/utils/ageEligibilityVerification.ts (hunk 2 above)
Comment (TatsyMaslova) [reply src/main/common/utils/ageEligibilityVerification.ts:4]: If claimant is 18 it should proceed as normal. I made changes and added the unit test for an 18 year old user. Thank you for your review!

**Review by taomoh — APPROVED** [review]

**Pull request merged**